extensions/resource-deployment/src/services/tools/azCliTool.ts

```diff
@@ -40,8 +43,86 @@ export class AzCliTool extends ToolBase {
 		return true;
 	}
 
-	public install(): Promise<void> {
-		return Promise.reject('not implemented');
+	protected get installationPath(): Promise<string | null> {
+		return new Promise<string | null>((resolve, _reject) => {
+			switch (this.osType) {
+				case OsType.linux:
+					resolve(defaultInstallationRoot);
+				default:
+					resolve(win32InstallationRoot);
+					console.log(`TCL: AzCliTool -> win32InstallationRoot:${win32InstallationRoot}`);
+			}
+		});
+	}
+	get installationCommands(): Command[] {
+		switch (this.osType) {
+			case OsType.darwin: return [
+				{
+					comment: `updating your brew repository ...`,
+					command: 'brew update'
```

Comment:
localize?

Reply:
I assume, you mean localize the comments, if so yes good call, I will.